dojo/urls.py

```diff
@@ -1,6 +1,5 @@
 from django.conf import settings
 from django.conf.urls import include, url
-from django.conf.urls.static import static
```

Comment:
This function is used in line 204 here. Was the deletion intentional?

Reply:
Hmm, no.